### DISCUSSION TITLE
App crashes when reloading the Metro server

### DISCUSSION THREAD

**Discussion by devsales** [software-mansion/react-native-reanimated#3639] [Q&A]
Importing Reanimated somewhere into the app causes it to crash when I reload the Metro server (press "r" in the terminal). This happens even if the component is not used anywhere, just the import statement is enough.

`import Animated, { SlideInRight } from 'react-native-reanimated'`

The animations work as expected, so there are no issues there. But the app crash is really annoying. Each time it produces a crash report I can send to apple (which of course I ignore) and then I have to start the app again in the simulator so that the Metro server can connect to it.

Has anyone else had this issue?

Platform: React Native 0.69.1 (react: 18.0.0)
Reanimated version: 2.10.0

**Comment by pfcodes** [comment]
Happens to me too
